diff --git a/build/common.gypi b/build/common.gypi
index 7150b3c..497b18f 100644
--- build/common.gypi
+++ build/common.gypi
@@ -1443,6 +1443,7 @@
     'ozone_platform_gbm%': 0,
     'ozone_platform_ozonex%': 0,
     'ozone_platform_test%': 0,
+    'ozone_platform_egl%': 0,
 
     # Chrome OS: whether to build ChromeVox from sources in the Chromium
     # repository rather than using precompiled JavaScript in
@@ -2214,6 +2215,7 @@
         'ozone_platform_dri%': 1,
         'ozone_platform_test%': 1,
         'ozone_platform_egltest%': 1,
+       'ozone_platform_egl%': 1,
       }],
 
       ['use_ozone==1 and ozone_auto_platforms==1 and chromeos==1', {
diff --git a/ui/ozone/ozone.gyp b/ui/ozone/ozone.gyp
index 65ab0a2..5775dcf 100644
--- ui/ozone/ozone.gyp
+++ ui/ozone/ozone.gyp
@@ -198,5 +198,10 @@
         'platform/test/test.gypi',
       ],
     }],
+    ['<(ozone_platform_egl) == 1', {
+      'includes': [
+        'platform/egl/egl.gypi',
+      ],
+    }],
   ],
 }
diff --git a/media/ozone/media_ozone_platform.cc b/media/ozone/media_ozone_platform.cc
index 804de37..fb351aa 100644
--- media/ozone/media_ozone_platform.cc
+++ media/ozone/media_ozone_platform.cc
@@ -40,6 +40,9 @@ MediaOzonePlatform* CreateMediaOzonePlatformEgltest() {
   return new MediaOzonePlatformStub;
 }
 
+MediaOzonePlatform* CreateMediaOzonePlatformEgl() {
+  return new MediaOzonePlatformStub;
+}
 MediaOzonePlatform* CreateMediaOzonePlatformGbm() {
   return new MediaOzonePlatformStub;
 }
